src/ORM/Table.php

```diff
@@ -460,17 +460,17 @@ public function schema($schema = null)
      * ### Example:
      *
      * ```
-     * protected function _initializeSchema(\Cake\Database\Schema\Table $table) {
+     * protected function _initializeSchema(\Cake\Database\Schema\TableSchema $table) {
      *  $table->columnType('preferences', 'json');
      *  return $table;
      * }
      * ```
      *
-     * @param \Cake\Database\Schema\Table $table The table definition fetched from database.
-     * @return \Cake\Database\Schema\Table the altered schema
+     * @param \Cake\Database\Schema\TableSchema $table The table definition fetched from database.
+     * @return \Cake\Database\Schema\TableSchema the altered schema
      * @api
      */
-    protected function _initializeSchema(Schema $table)
+    protected function _initializeSchema(TableSchema $table)
```

Comment:
Only if the typehints aren't changed in `Cake\ORM\Table`. Changing any of the typehints in commonly extended classes will cause strict errors in existing code.